Extensions/SystemInfo/systeminfotools.js

```diff
@@ -28,3 +28,17 @@ gdjs.evtTools.systemInfo.isMobile = function() {
 
 	return false;
 };
+
+/**
+ * The device supports WebGL
+ * @param {gdjs.RuntimeScene} runtimeScene
+ * @returns {Boolean} WebGL is supported
+ */
+gdjs.evtTools.systemInfo.isWebGLSupported = function(runtimeScene) {
+	var pixiRenderer = runtimeScene.getGame().getRenderer()._pixiRenderer;
```

Comment:
It's a good start, but we can do without having to dig into a private property of the renderer.
For this, let's add a new method into the PixiRenderer (and CocosRenderer), that is called isWebGLSupported.
From here, you just call it:
```suggestion
	return runtimeScene.getGame().getRenderer().isWebGLSupported();
```

And then you implement the method in both the Pixi renderer and Cocos2d-JS.
Here for Pixi: https://github.com/4ian/GDevelop/blob/master/GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js#L428
Here for Cocos2d-JS: https://github.com/4ian/GDevelop/blob/master/GDJS/Runtime/cocos-renderers/runtimegame-cocos-renderer.js#L121

You can see that `getCanvas` is following the same pattern: abstract the details in the renderer, so that you have one implementation for Pixi:
https://github.com/4ian/GDevelop/blob/33487fb0f69ab0f16fa3c8c0eefac3919484e05d/GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js#L425-L427
and one for Cocos2d-JS:
https://github.com/4ian/GDevelop/blob/33487fb0f69ab0f16fa3c8c0eefac3919484e05d/GDJS/Runtime/cocos-renderers/runtimegame-cocos-renderer.js#L118-L120

By doing this, we keep the game engine and events relatively agnostic of whatever is the game using to render itself on screen :) When implementing a new renderer, it's also easier: copy/paste an existing renderer and implement everything that is needed in this file - rather than having to poke around in the codebase, searching for "pixiRenderer" and add a new if/else branch for your new renderer.

Last question: how to implement this for Cocos2D-JS?? :D 
No obvious answer in the doc, the Cocos2d-JS documentation is quite lacking information and it's often hard to find what is the proper way to do things :/
There is a documented constant called cc.game.RENDER_TYPE_WEBGL but we have to compare it to the current render type which is stored in a undocumented `cc._renderType`. Not a big fan of doing it, but by looking at the [source](https://docs.cocos2d-x.org/api-ref/js/V3.13/symbols/src/_Users_jtsm_Chukong-Inc_cocos2d-x_web_CCBoot.js.html), I can't see anything better.

So for cocos2d-JS, it's: `return cc._renderType === cc.game.RENDER_TYPE_WEBGL;`. 
We could also choose not to implement this, but as it seems relatively ok, let's use this. If it breaks later (let's say that _renderType is renamed), then the condition will return false - not too bad.